src/Native/System.Security.Cryptography.Native/pal_ssl_client_cert.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_ssl_client_cert.h"
```

Comment:
The header and the implementation have a different order to the functions.  It doesn't really matter, but threw off my scanning.  Is there a reason they're different?